package.json

```diff
@@ -10,11 +10,11 @@
     "eslint": "eslint --report-unused-disable-directives .",
     "eslint:fix": "npm run eslint -- --fix",
     "test": "npm run test-unit; npm run test-integration;npm run test-repository",
-    "test-unit": "NODE_ENV=test mocha -r dotenv/config dotenv_config_path=.env.test --exit ./server/models/**/*.spec.js ./server/routes/**/*.spec.js",
-    "test-repository": "NODE_ENV=test mocha -r dotenv/config --exit --require co-mocha  ./server/repositories/**/*.spec.js",
+    "test-unit": "NODE_ENV=test mocha -r dotenv/config dotenv_config_path=.env.test --exit --require co-mocha ./server/models/**/*.spec.js ./server/routes/**/*.spec.js",
```

Comment:
I still don't get the meaning of this? what's the `--require` for?

Reply:
I think it works fine without it, I was just playing around with the scripts and kept it in for consistency with some others.